### PR TITLE
Miscellaneous Fixes for the Unfettered Campaign

### PR DESCRIPTION
**Pull request by implicitfield**
This includes various fixes to missions, grammar and comments.
This does break saves, since the names of some events and missions have been changed.

### CODE REVIEW

File: data/hai/unfettered 2 war preparations.txt [hunk 1]
```diff
@@ -356,15 +356,15 @@ mission "<uhai> 12: Outfitter Upgrade"
 		log `Helped the Unfettered upgrade their shipyards and received a prototype ship as thanks.`
 		event "unfettered: darkcloak outfitter upgrade" 30
 		conversation
-			`After unloading all the cargo, you go check for He-Yi in the shipyard, but instead you find an Herald, ready to rely a message to you, "Ah, <first>. You have completed your part of the bargain, now you may have your reward.`
-			`	"Go to Firelode, Mera has a very peculiar gift for you, should you prove yourself to be worthy of it.`
+			`After unloading all the cargo, you go check for He-Yi in the shipyard, but instead you find a herald, ready to rely a message to you, "Ah, <first>. You have completed your part of the bargain, now you may have your reward."`
+			`	"Go to Firelode, Mera has a very peculiar gift for you, should you prove yourself to be worthy of it."`
```
Comment (Hurleveur) [inline data/hai/unfettered 2 war preparations.txt:360]: Not including these " at the end is intentional when it is a multi paragraph quote, like it happens in HR (but to be honest I'm not sure of how it works)

File: data/hai/unfettered side missions.txt [hunk 2]
```diff
@@ -769,11 +769,12 @@ mission "<uhai> 20: Boarding Equipement: More Enforcer Staffs"
...
 	to offer
 		has "<uhai> 20: Bigeminate Sigil: Boarding Equipement: offered"
-		not "Bring Coalition Weapons: offered"
-		not "Bring Pug Staffs: offered"
+		not "<uhai> 20: Boarding Equipement: Enforcer Staffs: offered"
+		not "<uhai> 20: Boarding Equipement: Pug Staffs: offered"
 	on offer
 		require "Enforcer Riot Staff"
 		conversation "more staffs"
+		fail
```
Comment (Hurleveur) [inline data/hai/unfettered side missions.txt:777]: The convo ends with declining by default so this isnt required
Cant hurt for clarity tho

**Review by Hurleveur — APPROVED** [review]
Nice! Thx a lot for all of this good work, especially in the form of a PR!
Looks like I had forgotten to rename some mission triggers, woops

There is only one small thing (the " isnt needed for multi lined paragraph) but other than that very good fixes and improvements, so I'll merge it and adjust those quotes locally.

**Comment by Hurleveur** [comment]
Btw I'm interested in any feedback you may have on the campaign itself! We can talk about it here, on the main PR or in the discord, if you're interested.

**Comment by implicitfield** [comment]
Sure! I really enjoyed the campaign, thanks for writing it. It has a very unique feel to it compared to everything else in ES. What I found especially good about the campaign was:
- The amount of morally grey choices.
- The feeling of danger in many missions. (e.g. one wrong move and the player is down in the JD raid mission, and in general discouraging excessively "brave" actions.) The involvement of shadowy organizations definitely helps with this, and they add lots of flavor to the story.
  - As an extension to that, it's great that the player has to actually think about their choices, and truly has to prove their worth.
- The use of advanced / new features. Especially the use of conditional conversations helps immerse the player in the story, and so do the outfitter and shipyard missions, though I guess it isn't always obvious when those might happen.

As a sort of bug report, it seems like it's possible that the ```midas: took out leader``` condition gets set twice, if the player destroys the flagship in the ```Midas: Daemons Unleashed``` mission and in the later Ion Materials mission, though I didn't test that. I guess those missions should somehow adapt to that condition, or maybe failing the ```<uhai> 7: Duel``` mission should block or otherwise affect the rest of the campaign.

**Comment by Hurleveur** [comment]
> As a sort of bug report, it seems like it's possible that the midas: took out leader condition gets set twice, if the player destroys the flagship in the Midas: Daemons Unleashed mission and in the later Ion Materials mission, though I didn't test that. I guess those missions should somehow adapt to that condition, or maybe failing the <uhai> 7: Duel mission should block or otherwise affect the rest of the campaign.

That's intentional, the idea is that if you take out that ship they will no longer send mercenaries attacking you, but I will rename it to make it clearer that the condition means that. `midas: proven worthy` is probably better.
It's intended that you don't *have* to ever protect any npcs, failing that mission only cuts you from any future midas missions (and there will be some in the next part or so, probably as an independent PR)

**Comment by implicitfield** [comment]
Alright, that makes sense. It seemed confusing that you'd be able to take out the leader twice, but it totally works if the name of the condition is changed.

**Comment by Hurleveur** [comment]
The idea with the Nemo Me Impune Lacessit organization is you don't mess with them or they come kill you. (hence the name)
It happens if you annoy Si-Ra and Midas too much, and it happens if you kill one of the twins.

There probably will be a mission where they come to your ship to try and kill you with an elite team, too, if you did not do something severe enough to warrant the whole fleet but still pissed them quite a bit.

**Comment by implicitfield** [comment]
That seems quite interesting. Looking forward to that mission (and indeed, the rest of the campaign.)

**Pull request merged**